server/settings/conversion.py

```diff
@@ -71,10 +71,36 @@ def _convert_validate_version_0_3_3(publish_overrides):
     validate_version["plugin_state_profiles"] = [profile]
 
 
+def _convert_oiio_transcode_0_4_5(publish_overrides):
+    """ExtractOIIOTranscode plugin changed in 0.4.5."""
+    if "ExtractOIIOTranscode" not in publish_overrides:
+        return
+
+    transcode_profiles = publish_overrides["ExtractOIIOTranscode"]["profiles"]
+
+    for profile in transcode_profiles:
+        for output in profile["outputs"]:
+            transcode_type = output["transcoding_type"]
+            if transcode_type == "display":
+                output["transcoding_type"] = "display_view"
+            # Already new settings
+            if "display_view" in output:
+                continue
+
+            output["display_view"] = {}
+            if output["display"]:
+                output["display_view"].update({"display": output["display"]})
+                output.pop("display")
+            if output["view"]:
+                output["display_view"].update({"view": output["view"]})
+                output.pop("view")
+
```

Comment:
```suggestion
        outputs = profile.get("outputs")
        if outputs is None:
            break

        for output in outputs :
            # Already new settings
            if "display" not in output and "view" not in output:
                break

            # Fix 'display' -> 'display_view' in 'transcoding_type'
            transcode_type = output.get("transcoding_type")
            if transcode_type == "display":
                output["transcoding_type"] = "display_view"
            
            # Convert 'display' and 'view' to new values
            output["display_view"] = {
                "display":  output.pop("display", ""),
                "view": output.pop("view", ""),
            }

```

Reply:
That's why condition `if "display" not in output and "view" not in output:` is there.

Reply:
Lets merge the suggestion so this could be approved @MustafaJafar

Reply:
Sorry one more question. 
should we `continue` instead of `break` ?

```suggestion
        outputs = profile.get("outputs")
        if outputs is None:
            return

        for output in outputs :
            # Already new settings
            if "display" not in output and "view" not in output:
                continue

            # Fix 'display' -> 'display_view' in 'transcoding_type'
            transcode_type = output.get("transcoding_type")
            if transcode_type == "display":
                output["transcoding_type"] = "display_view"
            
            # Convert 'display' and 'view' to new values
            output["display_view"] = {
                "display": output.pop("display", ""),
                "view": output.pop("view", ""),
            }

```

Reply:
> should we continue instead of break ?

If one output does not have it, then none of them have it.